src/ship_cmd.cpp

```diff
@@ -66,7 +66,7 @@ bool IsValidImageIndex<VEH_SHIP>(uint8 image_index)
 	return image_index < lengthof(_ship_sprites);
 }
 
-static inline TrackBits GetTileShipTrackStatus(TileIndex tile)
```

Comment:
I do not think removing `inline` is a good choice here. More commonly, these functions are in a `.h` file.

Reply:
I followed the example set by WaterClass GetEffectiveWaterClass(TileIndex tile);

Reply:
Do you understand what `inline` does? And with that knowledge, do you still think it is a good idea to 'just remove' that? How is `GetEffictiveWaterClass` an example? Did you see a similar change there, where someone removed `inline`? And how does it relate to this function? I have so many questions.